internal/cli/atlas/streams/instance/create.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package instance
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli/require"
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/store"
+	"github.com/mongodb/mongodb-atlas-cli/internal/usage"
+	"github.com/spf13/cobra"
+	atlasv2 "go.mongodb.org/atlas-sdk/v20230201004/admin"
+)
+
+type CreateOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	cli.InputOpts
+	name     string
+	provider string
+	region   string
+	store    store.StreamsCreator
+}
+
+const (
+	createTemplate = "Atlas Streams Processor Instance '{{.Name}}' successfully created.\n"
+)
+
+func (opts *CreateOpts) Run() error {
+	streamProcessor := atlasv2.NewStreamsTenant()
+	streamProcessor.Name = &opts.name
+	streamProcessor.GroupId = &opts.ProjectID
+	streamProcessor.DataProcessRegion = atlasv2.NewStreamsDataProcessRegion(opts.provider, opts.region)
+
+	r, err := opts.store.CreateStream(opts.ProjectID, streamProcessor)
+
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+func (opts *CreateOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+// CreateBuilder
+// atlas streams instance create [name]
+// --provider AWS
+// --region VIRGINIA_USA.
+func CreateBuilder() *cobra.Command {
+	opts := &CreateOpts{}
+	cmd := &cobra.Command{
+		Use:   "create [name]...",
```

Comment:
can you pass multiple names? as in
```bash 
atlas instance create instanceA instanceB
```
?

Reply:
reading the code seems like only one name is valid
```suggestion
		Use:   "create [name]",
```

Reply:
Was resolved in [1cb5065](https://github.com/mongodb/mongodb-atlas-cli/pull/2138/commits/1cb5065fed04f4401642cd750a2e1d0c6e4622cb)